src/middleware/Middleware.sol

```diff
@@ -345,7 +352,6 @@ contract Middleware is SimpleKeyRegistry32, Ownable {
         if (params.totalOperatorStake < amount) {
             revert Middleware__TooBigSlashAmount();
         }
-
         // simple pro-rata slasher
         for (uint256 i; i < s_vaults.length(); ++i) {
```

Comment:
maybe change uint256 to smaller type as there is no way there will be 115792089237316195423570985008687907853269984665640564039457584007913129639935 vaults connected to tanssi. find something that maxes at 1024

Reply:
If there's not a particular reason to decrease it, it's better to leave uint256 as it is costs less gas